src/middlewares/noAuth.middleware.js

```diff
@@ -0,0 +1,23 @@
+const { ApiError } = require('../exceptions/api.error');
+const jwtService = require('../services/jwt.service');
+
+const noAuthMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
+
+  if (authorization || token) {
```

Comment:
The condition `if (authorization || token)` will always be true if `authorization` is not an empty string, even if `token` is undefined. This might lead to unintended behavior. Consider checking `if (authorization && token)` instead.